js/collapse.js

```diff
@@ -16,7 +16,7 @@
   var Collapse = function (element, options) {
     this.$element      = $(element)
     this.options       = $.extend({}, Collapse.DEFAULTS, options)
-    this.$trigger      = $(this.options.trigger).filter('[href="#' + element.id + '"], [data-target="#' + element.id + '"]')
+    this.$trigger      = $('[data-toggle="collapse"]').filter('[href="#' + element.id + '"], [data-target="#' + element.id + '"]')
```

Comment:
But it would need to be `'[data-toggle="collapse"][href="' + element.id '"], [data-toggle="collapse"][data-target="#' + element.id +  '"]'` which is longer.